src/category_theory/sites/limits.lean

```diff
@@ -166,7 +166,7 @@ end limits
 section colimits
 
 universes w v u
-variables {C : Type (max v u)} [category.{v} C] {J : grothendieck_topology C}
+variables {C : Type u} [category.{v} C] {J : grothendieck_topology C}
 variables {D : Type w} [category.{max v u} D]
```

Comment:
Should this be `u` instead of `max v u` too?

Reply:
My understanding is that changing this would involve a larger refactor (see https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/Universe.20issues.20with.20concrete.20categories/near/350344976 ) i.e. adding new lemmas and removing old ones (which I'm not really qualified to do, I've never thought too hard about this universe nonsense), and my intention was just to do something which made the port easier.